openvdb/points/AttributeArray.h

```diff
@@ -169,6 +205,37 @@ class OPENVDB_API AttributeArray
     /// @note This counts each data element in a strided array
     virtual Index dataSize() const = 0;
 
+#if OPENVDB_ABI_VERSION_NUMBER >= 6
+    /// Return the name of the value type of a single element in this array (e.g., "float" or "vec3d").
+    virtual Name valueType() const = 0;
+
+    /// Return the name of the codec used by this array (e.g., "trnc" or "fxpt").
+    virtual Name codecType() const = 0;
+
+    /// Return the size in bytes of the value type of a single element in this array.
+    /// (e.g. "float" -> 4 bytes, "vec3d" -> 24 bytes").
+    virtual Index valueTypeSize() const = 0;
+
+    /// Return the size in bytes of the storage type of a single element of this array.
+    /// @note If the Codec is a NullCodec, valueSize() == storageSize()
+    virtual Index storageTypeSize() const = 0;
+
+    /// Return @c true if the value type is floating point
+    virtual bool valueTypeIsFloatingPoint() const = 0;
+
+    /// Return @c true if the value type is scalar (not vector, quaternion or matrix)
```

Comment:
The comment is the wrong way round here too

Reply:
Thanks, just resolved that.